Sources/Defaults/Defaults+Bridge.swift

```diff
@@ -406,14 +406,42 @@ extension Defaults {
 			guard let value = value else {
 				return nil
 			}
+			guard
+				let cgColor = value.cgColor,
+				let colorSpace = cgColor.colorSpace?.name as? String,
+				let components = cgColor.components
+			else {
+				return NativeColor.bridge.serialize(NativeColor(value))
+			}
 
-			return NativeColor.bridge.serialize(NativeColor(value))
+			return [colorSpace, components]
 		}
 
 		public func deserialize(_ object: Serializable?) -> Value? {
-			guard let nativeColor = NativeColor.bridge.deserialize(object) else {
+			if let object = object as? NativeColor.Serializable {
+				guard let nativeColor = NativeColor.bridge.deserialize(object) else {
+					return nil
+				}
+
+				return Value(nativeColor)
+			}
+
+			guard
```

Comment:
Can you add a short comment that says why we do this. Because `UIColor` does not hold a color space, but `Swift.Color` does. Just for any readers wondering why we made it so complicated.

Reply:
Updated! Thanks for the reminder!